### ISSUE TITLE
new channel list does not appear

### ISSUE THREAD

**Issue by andygavr** [PredatH0r/ChanSort#202]
Hello 
I did a new list with ChanSort but when I load it into my new TV (LG OLED65CX9) only a few channels appear
Most channels I cannot see
the list changes from the old one but only 5% of the new channels are visible
what can this be?
I tried to load the list several times

**Comment by PredatH0r** [comment]
There is a really long thread about LG's webOS 5 (and specifically the CX series): https://github.com/PredatH0r/ChanSort/issues/172

TL;DR:
One user has a CX model with Firware 3.11 and DVB-C, which seems to work in ChanSort when the channels were searched with "Provider: other" 
Another user has a model with firmware version 5.xxx and DVB-S and that does not seem to work.

I have no idea what models and/or firmware versions and/or input sources (DVB-C/T/S) actually work or don't work.
My current impression is that webOS is in most cases simply incapable of importing lists - at least when there are already channels in the TV. Maybe (and that is unconfirmed) the import works after resetting the TV and clearing the internal channel list.

**Comment by PredatH0r** [comment]
It seems that webOS 5 related issues are fixed when using recent versions of LG firmware and ChanSort.
